src/js/components/More/PricingCard.jsx

```diff
@@ -108,202 +111,456 @@ class PricingCard extends Component {
                 <br />
                 <Collection>
                   <CollectionItem>
-                    <Check
-                      style={{
-                        color: '#2E3C5D',
-                      }}
-                    />
-                    <ItemText>Enter Your Own Positions</ItemText>
+                    {pricingCardFeatures[0].iconType === 'checkMark' ? (
```

Comment:
Hi @jarodpeachey, in order to reduce duplicate code, I was hoping you could create a for/loop for the CollectionItem, so that you could pass in "pricingCardFeatures" and it could loop through all of the data in that array. This would avoid hard-coding "ItemText" values in this file too.